### ISSUE TITLE
[Biometric] BiometricDemoD11.dproj double fires result handlers 

### ISSUE THREAD

**Issue by rj1102** [DelphiWorlds/Kastri#149]
**Description**
In the BiometricDemoD11.dproj the VerificationSuccessResultHandler is fired twice on a successfull verification of finger or face.
Also the VerificationFailResultHandler is hit twice on a failure (e.g. if you cancel the native finger print dialog).
Expected: it should fire only once.

**Error messages**
N/A

**Using**
Delphi 11.2 (Professional with Mobile) in Windows 11
Android SDK 25.2.5 - NDK r21
Testing on OPPO mobile with Android 12
Deploying android 32-bit

**Steps to Reproduce**
Run Kastri-master/Demos/Biometric/Biometric/BiometricDemoD11.dproj
Hit touch icon
Use finger for touch: VerificationSuccessResultHandler fires twice (expected: once); this also happens for face recognition
Hit Cancel button of biometric finger dialog: VerificationFailResultHandler fires twice (expected: once)

**Note**
In order to build and deploy the demo I had to change/remove some unavailable jar libraries, see below. Maybe something wrong there? 

activity-1.1.0.jar => remove, and add androidx-activity-1.3.1.jar
appcompat-1.2.0.jar => remove, and add androidx-appcompat-1.0.2.jar
appcompat-resources-1.2.0.jar => remove (no alternative found)
biometric-1.1.0.jar => remove, and add androidx-biometric-1.1.0.jar
BiometricDemoD11R.jar => remove (no alternative found)
fragment-1.2.5.jar => remove, and add fragment-1.3.6.jar
lifecycle-common-2.2.0.jar => remove, and add lifecycle-common-2.3.1.jar
lifecycle-livedata-core-2.2.0.jar => remove, and add lifecycle-livedata-core-2.3.1.jar
lifecycle-runtime-2.2.0.jar => remove, and add lifecycle-runtime-2.3.1.jar
lifecycle-viewmodel-savedstate-2.2.0.jar => remove, and add lifecycle-viewmodel-savedstate-2.3.1.jar
savedstate-1.0.0.jar => remove, and add androidx-savedstate-1.1.0.jar
vectordrawable-1.1.0.jar => remove (no alternative found)

I also tried the biometric demo in the Playground, but that won't deploy.

**Comment by DelphiWorlds** [comment]
> I also tried the biometric demo in the Playground, but that won't deploy.

Please give more details, e.g. any error messages

**Comment by rj1102** [comment]
> > I also tried the biometric demo in the Playground, but that won't deploy.
> 
> Please give more details, e.g. any error messages

I have opened a new issue for it #151

**Comment by rj1102** [comment]
This is fixed in the playground version of the biometric demo
